tools/cudaq-qpud/nvqc_proxy.py

```diff
@@ -53,7 +53,9 @@ def build_command_list(temp_file_name: str) -> list[str]:
     if WATCHDOG_TIMEOUT_SEC > 0:
         cmd_list = ['timeout', str(WATCHDOG_TIMEOUT_SEC)] + cmd_list
     if RUN_AS_NOBODY:
-        cmd_list = ['su', '-s', '/bin/bash', 'nobody', '-c', ' '.join(cmd_list)]
+        cmd_list = [
+            'su', '-s', '/bin/bash', 'nobody', '-c', ' '.join(cmd_list)
+        ]
```

Comment:
I think the formatter isn't happy with this change because I think you need to run `yapf -i --style google tools/cudaq-qpud/nvqc_proxy.py`.